Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -127,6 +130,17 @@ private void OnShutdown(EntityUid uid, ActiveBiomassReclaimerComponent component
             _ambientSoundSystem.SetAmbience(uid, false);
         }
 
+        private void OnPowerChanged(EntityUid uid, BiomassReclaimerComponent component, ref PowerChangedEvent args)
+        {
+            if (args.Powered)
+            {
+                if (component.ProcessingTimer > 0)
+                    _entityManager.EnsureComponent<ActiveBiomassReclaimerComponent>(uid);
+            }
+            else
+                RemCompDeferred<ActiveBiomassReclaimerComponent>(component.Owner);
```

Comment:
does this need to be deferred? can't you just use `RemComp`?

Reply:
I don't know surly how ticks work here, but if it will be more right to do - I change it.